internal/controller/scenario/scenario_adapter.go

```diff
@@ -53,6 +53,24 @@ func NewAdapter(context context.Context, application *applicationapiv1alpha1.App
 	}
 }
 
+// TODO: Remove after a couple weeks. This is just to add new field to existing scenarios
+// Adds ResolverType() field to existing IntegrationTestScenarios
+func (a *Adapter) EnsureScenarioContainsResolverType() (controller.OperationResult, error) {
+	a.logger.Info("Adding ResolverType to ITS if it does not exist", "scenario", a.scenario)
+	if a.scenario.Spec.ResolverRef.ResolverType == "" {
+		// set ResolverType to 'pipeline'
+		patch := client.MergeFrom(a.scenario.DeepCopy())
+		a.scenario.Spec.ResolverRef.ResolverType = "pipeline"
```

Comment:
I'm guessing this field name will need to be changed to `ResourceKind` after #1112 ?